hoot-services/src/main/java/hoot/services/controllers/grail/GrailResource.java

```diff
@@ -331,7 +331,7 @@ public Response createChangeset(@Context HttpServletRequest request,
                 differentialParams.setConflationType(DbUtils.getConflationType(Long.parseLong(input2)));
                 input2 = HOOTAPI_DB_URL + "/" + input2;
 
-                grailCommandClass = replacement ? DeriveChangesetReplacementCommand.class : DeriveChangesetCommand.class;
+                grailCommandClass = replacement ? DeriveChangesetReplacementCommand.class : RunDiffCommand.class;
```

Comment:
Trying to understand this change.  Was DeriveChangesetCommand part of conditional never invoked until now?

Reply:
it was for derive changeset but it never was able to produce the tags file before